### ISSUE TITLE
Exception on git queue-pop

### ISSUE THREAD

**Issue by losipiuk** [kokosing/git-gifi#18]
```
[eiger:~/workspace/presto/presto-main] (decimal-wip %) ⌘ git queue-pop
Traceback (most recent call last):
  File "/Users/losipiuk/workspace/virtualenvs/td/bin/gifi", line 11, in <module>
    sys.exit(main())
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/gifi/main.py", line 69, in main
    _main(args)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/gifi/main.py", line 76, in _main
    result = command(*args)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/gifi/command.py", line 37, in __call__
    return self.commands.get(commandName)(*commandArgs)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/gifi/command.py", line 37, in __call__
    return self.commands.get(commandName)(*commandArgs)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/gifi/command.py", line 9, in __call__
    return self.callable(*args, **kwargs)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/gifi/queue.py", line 11, in _pop
    repo.git.stash('apply', '--index')
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/git/cmd.py", line 440, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/git/cmd.py", line 834, in _call_process
    return self.execute(make_call(), **_kwargs)
  File "/Users/losipiuk/workspace/virtualenvs/td/lib/python2.7/site-packages/git/cmd.py", line 627, in execute
    raise GitCommandError(command, status, stderr_value)
git.exc.GitCommandError: 'git stash apply --index' returned with exit code 1
stderr: 'error: patch failed: presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java:385
error: presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java: patch does not apply
error: patch failed: presto-main/src/main/java/com/facebook/presto/type/DecimalOperators.java:52
error: presto-main/src/main/java/com/facebook/presto/type/DecimalOperators.java: patch does not apply
Conflicts in index. Try without --index.'
```

**Comment by kokosing** [comment]
There are conflicts in your patch.

**Comment by kokosing** [comment]
It is not yet supported.

**Comment by kokosing** [comment]
fixed in 0.10 version.
